spec/rails_helper.rb

```diff
@@ -54,8 +54,6 @@
     Aws.config[:stub_responses] = true
 
     ENV['GITHUB_ORGANIZATION'] = 'degica'
-    ENV['VAULT_URL'] ||= "http://my-vault.com"
```

Comment:
removed this because if this env is set the vault auth is used for all test cases. we want to use GitHub auth except for test cases that tests vault auth